.github/CODEOWNERS

```diff
@@ -24,3 +24,4 @@
 /packages/tools/integration-tests               @ggobugi27
 /packages/tools/remark-plugins                  @webpro
 /packages/tools/rush-fix-versions               @alber70g @webpro
+/packages/apps/immutable-records                @barthuijgen
```

Comment:
Please remove this or add someone (like @webpro), since you can't approve your own PR.

Reply:
Approving your own PR is not the purpose of CODEOWNERS, it's to approve when someone else makes changes to that code right? But I'll remove it.